include/wx/fontutil.h

```diff
@@ -183,6 +183,7 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
 public :
 #elif defined(__WXQT__)
     QFont m_qtFont;
+    int m_wxFontWeight;
```

Comment:
_If_ we decide to keep it (see the general comment), I think this should be just called `m_weight` for consistency with the other places (everything in "wx" is implicitly "wx" unless it explicitly isn't, as `m_qtFont` above), but most importantly it would be really nice to have a comment here explaining that we need to keep the weight here because the conversion between wx and Qt weight is not lossless.

Reply:
I will apply this suggestion if we keep this member, however I will probably look to apply your suggestion in the general comment about removing this member and making the tests a little more forgiving when checking the returned numeric weight.